### ISSUE TITLE
The 'qonversion_flutter_updated_entitlements' channel sent a message from native to Flutter on a non-platform thread.

### ISSUE THREAD

**Issue by LorTos** [qonversion/flutter-sdk#339]
When using `Qonversion.getSharedInstance().updatedEntitlementsStream` to listen to entitlement updates, I receive this message in the console.

```
[ERROR:flutter/shell/common/shell.cc(1055)] The 'qonversion_flutter_updated_entitlements' channel sent a message from native to Flutter on a non-platform thread. Platform channel messages must be sent on the platform thread. Failure to do so may result in data loss or crashes, and must be fixed in the plugin or application code creating that channel.
See https://docs.flutter.dev/platform-integration/platform-channels#channels-and-platform-threading for more information.
```

**Comment by suriksarkisyan** [comment]
Hi, @LorTos 
Thanks for the details! Which platform do you face this issue? iOS, Android, or both?

**Comment by LorTos** [comment]
Seems to be happening only on iOS

**Comment by suriksarkisyan** [comment]
Thanks for the reply. We will fix it in the next release.

**Comment by LorTos** [comment]
Thanks!

**Comment by suriksarkisyan** [comment]
Hi @LorTos 
I am happy to say that we released the new version [9.1.0](https://github.com/qonversion/flutter-sdk/releases/tag/9.1.0), which includes a fix related to this issue!
